CHANGELOG.md

```diff
@@ -1,3 +1,3 @@
-## 0.0.1-dev
+## 0.0.2-dev
```

Comment:
I'd go with 0.1.0, honestly

Strictly speaking, 0.0.2 is a COMPATIBLE release in the `0.0` prerelease series.